pkgs/by-name/gh/gh-copilot/package.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, fetchurl
+, system
+}:
+let
+  systemToBinary = {
+    "x86_64-linux" = {
+      name = "linux-amd64";
+      hash = "12mysfa84fzq3f95mi6i46yrcbdl3y91sn53snr4qqhm2w8yzb0l";
+    };
+    "aarch64-linux" = {
+      name = "linux-arm64";
+      hash = "14yd5h7fncq2aamxvcfjb5wsi64gs16gs2zj6gihg9ifglxgvxg2";
+    };
+    "x86_64-darwin" = {
+      name = "darwin-amd64";
+      hash = "0888gis7f2frziiskrws9fzfjshzydr14j8kapfqx8qcgcljb0av";
+    };
+    "aarch64-darwin" = {
+      name = "darwin-arm64";
+      hash = "1whywjjdzppswij3hrq3avgyhl363c4l5a4x5lixk9kzm3mq0qqp";
+    };
+  };
```

Comment:
This should have an or unsupported system to eval on other platforms.

see https://github.com/NixOS/nixpkgs/blob/4a9a5fab1c2db1ede39ab3877c597f8ff4cae247/pkgs/applications/editors/vscode/vscode.nix#L20

You don't need to assign it to a variable here.

Reply:
Done